docs/fundamentals/custom-resource-urls.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Define custom resource URLs
+description: Learn how to create custom URLs for .NET Aspire resources.
+ms.date: 04/08/2025
+ms.topic: how-to
+---
+
+# Define custom resource URLs
+
+.NET Aspire resources that expose endpoints only configure host and port values. However, there might be situations where you want to access a specific path of an exposed endpoint. The host and port are unknown until run time. In these cases, you can use custom resource URLs to define specific paths on a configured endpoint, which is convenient for accessing resources from the [dashboard](dashboard/overview.md).
```

Comment:
Would be good to mention here that you can also just add URLs to any resource, not related to endpoints at all.